.github/workflows/check.yml

```diff
@@ -74,6 +74,14 @@ jobs:
           RUST_LOG=trace cargo +${{ matrix.rust-toolchain }} llvm-cov nextest $BUILD_TYPE --no-fail-fast --lcov --output-path lcov.info
           cargo +${{ matrix.rust-toolchain }} bench --no-run
 
+      - name: Run tests with sanitizers
+        if: matrix.os == 'ubuntu-latest' && matrix.rust-toolchain == 'nightly'
```

Comment:
I guess sanitizers only work on Linux?

Reply:
I am not sure. Looking into it now. 

https://doc.rust-lang.org/stable/unstable-book/compiler-flags/sanitizer.html

Reply:
All sanitizers are available on Linux and some on MacOS. I don't know how to run any of them on Windows.